### ISSUE TITLE
404 error when Accept header not set

### ISSUE THREAD

**Issue by fela** [MyProfile/myprofile#20]
I was getting 404 errors when trying to read profile cards with RDF.rb, it took me a little to understand why with web browsers it was working correctly instead. I found that the server returns a 404 error with clients not using content negotiation. Shouldn't it use a default fall back?

```
$ curl -I https://my-profile.eu/people/test/card#me
HTTP/1.1 404 Not Found
Date: Tue, 28 Aug 2012 12:51:36 GMT
Server: Apache/2.2.16 (Debian)
Vary: Accept-Encoding
Content-Type: text/html; charset=iso-8859-1


$ curl -I -H "Accept: application/rdf+xml" https://my-profile.eu/people/test/card#me
HTTP/1.1 303 See Other
Date: Tue, 28 Aug 2012 12:51:20 GMT
Server: Apache/2.2.16 (Debian)
Location: https://my-profile.eu/people/test/foaf.rdf
Vary: Accept-Encoding
Content-Type: text/html; charset=iso-8859-1
```

**Comment by deiu** [comment]
Thanks for reporting this. I'll look into it asap.

**Comment by deiu** [comment]
Fixed in 00cdd88a626e1e489502d8a4c876e048be4aff17 ("develop" branch).
